src/Peering/Peering/Peering.csproj

```diff
@@ -12,7 +12,8 @@
 
   <ItemGroup>
     <PackageReference Include="AutoMapper" Version="6.2.2" />
-    <PackageReference Include="Microsoft.Azure.Management.Peering" Version="0.9.0-preview" />
+    <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="2.3.20" />
```

Comment:
@rileymckenna same comment about removing this dependency since it's already defined in the `.targets` file